README.MD

```diff
@@ -216,7 +222,10 @@ sudo ifdown <b>eth1</b> && sudo ifup <b>eth1</b>
 
 ### 2.4.2 DHCP server
 
-A service called <b>dnsmasq</b> has the required dhcp server so first install dnsmasq:
+DHCP server is used to allow a host such as BBB to obtain the required TCP/IP configuration.
```

Comment:
DHCP server is used to give IP addresses to devices in a network. Also the only relevant thing here is that DHCP provides the IP address.